src/test/java/SalesforceModules/InvoiceUtil.java

```diff
@@ -229,7 +229,68 @@ public static void paymentsBilling(String account,String amount,String paymentty
         selectInvoices(invoice,"select invoice");
         KeywordUtil.takeScreenshotAndAttachInReport();
         QuoteUtil.clickSaveButton("clicked save button");
+    }
+
+    /**
+     * Select the SKU for the options
+     *
+     * @param logStep the log
+     * @param optionalSKU the optionalSKU
+     */
+    public static void selectOptionalSKU(String optionalSKU, String logStep) throws InterruptedException {
+        boolean flag = false;
+        KeywordUtil.waitForVisible(InvoiceObject.selectOptionalSKU);
+        KeywordUtil.click(InvoiceObject.selectOptionalSKU, logStep);
+        KeywordUtil.inputText(InvoiceObject.selectOptionalSKU, "Sam", logStep);
+        KeywordUtil.delay(5000);
+        Thread.sleep(5000);
+        String accname = "//lightning-base-combobox-formatted-text[contains(@title,'Samsung')]";
```

Comment:
Why we are using the xpath here directly if it is not dynamic?

Reply:
done